Jenkinsfile.deploy

```diff
@@ -4,7 +4,7 @@ helper = null
 
 pipeline {
     agent {
-        label 'kogito-operator-slave && !master'
+        label 'kogito-operator-replica && !main'
```

Comment:
this needs to be aligned with @radtriste

Reply:
This cannot be changed as it is Jenkins direct configuration. It should be done as part of another PR as it could have some side effects if changing right now the Jenkins node name

Reply:
And this goes for all Jenkinsfile ;)